cmd/root.go

```diff
@@ -47,9 +47,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
-type VNextJson struct {
-	Blobfuse2        string              `json:"blobfuse2"`
-	SecurityWarnings map[string][]string `json:"securityWarnings"`
+type ContainerList struct {
```

Comment:
I think this represents file list and not container list.

Reply:
Renamed to VersionFilesList